templates/gig/gig_form.html

```diff
@@ -219,7 +219,7 @@
             <div class="form-group">
                 <div class="form-check">
                     <input class="form-check-input" type="checkbox" id="newgig_notifymembers" name="gig_notifymembers" 
-                    {% if band.send_updates_by_default %}checked{% endif %}>
+                    {% if email_members %}checked{% endif %}>
```

Comment:
Maybe I'm missing something but I can't see what this variable is referencing?

Reply:
Nothing - it's in a section that's commented out! I was experimenting before I noticed. I'll remove the whole commented-out section.